src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetManagedTemplatePackage.cs

```diff
@@ -81,6 +83,23 @@ internal class NuGetManagedTemplatePackage : IManagedTemplatePackage
             _logger = settings.Host.LoggerFactory.CreateLogger<NuGetInstaller>();
         }
 
+        public string? Verified
+        {
+            get => Details.TryGetValue(VerifiedKey, out string verified) ? verified : "✘";
+
+            set
+            {
+                if (string.IsNullOrWhiteSpace(value))
+                {
+                    Details.Remove(VerifiedKey);
+                }
+                else
+                {
+                    Details[VerifiedKey] = bool.TryParse(value, out bool verified) && verified ? "✔" : "✘";
```

Comment:
please consider keeping it as true or false and leave final representation to the host.
For example, VS might want to use different representation and custom symbols are hard to parse.